service/controller/resource/promremotewrite/create.go

```diff
@@ -0,0 +1,75 @@
+package promremotewrite
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/v7/pkg/controller/context/resourcecanceledcontext"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	r.logger.Debugf(ctx, "reconcile remotewrite")
```

Comment:
Trying to follow the logging message pattern from [giantswarm/fmt](https://github.com/giantswarm/fmt/blob/master/go/log_messages.md), I would recommend something like :

```suggestion
	r.logger.Debugf(ctx, "ensuring prometheus remoteWrite config")
```